src/modules/flow/http-client/http-client.json

```diff
@@ -0,0 +1,195 @@
+{
+  "$schema": "http://solettaproject.github.io/soletta/schemas/node-type-genspec.schema",
+  "name": "http-client",
+  "meta": {
+    "author": "Intel Corporation",
+    "license": "BSD 3-Clause",
+    "version": "1"
+  },
+  "types": [
+    {
+      "category": "output/network",
+      "description": "HTTP Client for boolean",
+      "methods": {
+        "close": "common_close",
+        "open": "boolean_open"
+      },
+      "name": "http-client/boolean",
+      "options": {
+        "members": [
+          {
+            "data_type": "string",
+            "description": "The http url",
+            "name": "url"
+          }
+        ],
+        "version": 1
+      },
+      "in_ports": [
+	{
+	  "data_type": "any",
+	  "description": "Get the contents of the specified url",
+          "methods": {
+            "process": "boolean_get_process"
+          },
+          "name": "GET"
+        },
+	{
+	  "data_type": "boolean",
+	  "description": "Post into the url",
+          "methods": {
+            "process": "boolean_post_process"
+          },
+          "name": "POST"
+        }
```

Comment:
add an `URL` port for all nodes... it's simple and will allow you to change the URL during runtime (ie: from persistence nodes)